server/pkg/kvdb/LICENSE

```diff
@@ -0,0 +1,19 @@
+Copyright (C) 2015-2022 Lightning Labs and The Lightning Network Developers
```

Comment:
I assume was hard to use badger or sqlite directly?

Reply:
This package is a copy/paste of the [lnd one](https://github.com/lightningnetwork/lnd/tree/master/kvdb).

I simplified it by removing some impl, they already have the sqlite one, therefore it should be possible to use that kind of db. I'm gonna apply the changes

Reply:
the sqlite db depends on [sqldb](https://github.com/lightningnetwork/lnd/tree/master/sqldb) module which is a tailored db for the lnd node (check inside the [sqlc folder](https://github.com/lightningnetwork/lnd/tree/master/sqldb/sqlc/queries)). 

We'll add other macaroon dbs later